test/e2e/framework/pods.go

```diff
@@ -259,3 +260,27 @@ func (c *PodClient) MatchContainerOutput(name string, containerName string, expe
 	}
 	return nil
 }
+
+// MatchContainerLastLineOfOutput gets last line of output of a container and match expected regexp in the output.
+func (c *PodClient) MatchContainerLastLineOfOutput(name string, containerName string, expectedRegexp string) error {
```

Comment:
Probably can be removed. Can we check if there is a way to exec commands in containers in node e2e environments?

Reply:
`KubectlCmd` doesn't work out of the box.